source/installing-splunk/splunk-app.rst

```diff
@@ -67,4 +67,71 @@ Installation
     :align: center
     :width: 100%
 
-Now that you've finished installing Splunk app for Wazuh, you can install and setup Splunk forwarders on the :ref:`next section <splunk_forwarder>`.
+Now that you've finished installing Splunk app for Wazuh, you can install and setup Splunk forwarders on the :ref:`Splunk forwarder section <splunk_forwarder>`.
+
+Installing the Wazuh App in a Splunk cluster
+--------------------------------------------
+
+.. note::
+  We can install the App in each search-head by hand, but if we have hundreds or even thousands of search-heads, it will be better to install it automatically.
+  For this purpose, we are using the **deployer**, a machine that installs the App in every search-head at the same time and automatically.
+
+
+.. warning::
+  We need to eliminate "SplunkAppForWazuh/default/indexes.conf" so it does not create automatic indexes in the search-heads.
```

Comment:
This is not used anymore since we don't include the `indexes.conf` file in the app.